### PR TITLE
Fix kubernetes build directory support error

### PR DESCRIPTION
**Pull request by chenyb4**
Fix code not support 'kubernetes' or 'federation'
directory. Fix comment "Scenario should only run from either kubernetes
or federation directory!" code error.

Signed-off-by: Yuanbin.Chen <cybing4@gmail.com>

### CODE REVIEW

File: scenarios/kubernetes_build.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def main(args):
 
     This is a python port of the kubernetes/hack/jenkins/build.sh script.
     """
-    if os.path.split(os.getcwd())[-1] != 'kubernetes' and \
+    if os.path.split(os.getcwd())[-1] != 'kubernetes' or \
```
Comment (BenTheElder) [inline scenarios/kubernetes_build.py]: This change doesn't make sense. This will always error since the path can't be both..?

**Comment by chenyb4** [comment]
@BenTheElder
I want known both directory support is right or both diretory only one exist is right?
I see the comment tell me only one is right.
code comment:
'Scenario should only run from either kubernetes or federation directory!'

**Comment by BenTheElder** [comment]
/close

feel free to /reopen if this is still needed... I think this is incorrect though

**Pull request closed without merging**